student-blogs/BandyNaranjo-5656/styles/base.scss

```diff
@@ -0,0 +1,70 @@
+@use "variables.scss" as variables;
+
+body {
+    margin: 100px auto;
+    max-width: 800px;
+    font-family: "Helvetica", "Arial", sans-serif;
+    line-height: 1.5;
+    padding: 0 40px;
+    background-color: variables.$principal-color;
+}
+
+p a {
+    color: #333;
+}
+
+h2, h3 {
+    margin-top: 1em;
+    padding-top: 1em;
+    text-align: center;
+}
+
+h1, h2 {
+    color: white;
+}
+    
+strong {
+    color: #333;
+}
+
+.hobby {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
```

Comment:
Excellent work done, just as a suggestion when working with Sass one of its advantages is to use nesting. For example you can use it inside the `.hobby` class, it would look like this: 

`.hoddy {
display: flex;
flex-direction: column;
align-items: center;

p {
styles
}

img {
styles
}

}`